.gitignore

```diff
@@ -13,6 +13,9 @@
 # HTML coverage generated by gocov
 cover.html
 
+# JUnit XML
+*junit.xml
```

Comment:
could we not get all these files (including cover.html) generated somewhere under build? I think it is much nicer to generate all artefacts, be it binaries or test results, under a common base directory. If you want to clean up, you just delete the build directory.